client/python/lib/test/test_grpc_requests.py

```diff
@@ -75,3 +78,52 @@ def test_make_metadata_request_invalid(mocker, name, version, expected_exception
     
     assert str(e_info.value) == expected_message
     mock_method.assert_called_once()
+
+@pytest.mark.parametrize("inputs, expected_proto, name, version", PREDICT_REQUEST_VALID)
+def test_make_predict_request_valid(inputs, expected_proto, name, version):
+    model_predict_request = make_predict_request(inputs ,name, version)
+    
+    assert isinstance(model_predict_request, GrpcPredictRequest)
+    assert model_predict_request.model_name == name
+    assert model_predict_request.model_version == version
+    assert model_predict_request.inputs == inputs
+    assert isinstance(model_predict_request.raw_request, PredictRequest)
```

Comment:
Raw grpc request instance is used multiple times. Can you assign it to a new variable like:
`raw_predict_request = model_predict_request.raw_request` to make all next calls a bit shorter?